app/views/NewServerView/index.js

```diff
@@ -105,11 +121,16 @@ class NewServerView extends React.Component {
 	}
 
 	setHeader = () => {
-		const { adding, navigation } = this.props;
-		if (adding) {
+		const { adding, navigation, root } = this.props;
+		if (adding && root === ROOT_NEW_SERVER) {
 			navigation.setOptions({
+				headerTitle: I18n.t('Workspaces'),
 				headerLeft: () => <HeaderButton.CloseModal navigation={navigation} onPress={this.close} testID='new-server-view-close' />
 			});
+		} else {
```

Comment:
More like a taste than everything else, but we don't use `else` that much.
On this case, you could've done with:
```
if () {
  return A
}
return B
```